BringgSDK.js

```diff
@@ -22,14 +22,18 @@ var BringgSDK = (function () {
     "ew1": "https://eu1-realtime.bringg.com/",
     "ue1": "https://realtime2-api.bringg.com/",
     "ew1g" : "https://eu2-realtime.bringg.com/",
-    "ue4g": "https://us2-realtime.bringg.com/"
+    "ue4g": "https://us2-realtime.bringg.com/",
+    "ew3g": "https://eu3-admin-api.bringg.com",
+    "uc1g": "https://us3-admin-api.bringg.com",
```

Comment:
Also please change `eu1` to `https://realtime.bringg.com/`

```suggestion
    "ew3g": "https://eu3-realtime.bringg.com/",
    "uc1g": "https://us3-realtime.bringg.com/",
```